### DISCUSSION TITLE
Add `--no-typescript` to `create-astro`

### DISCUSSION THREAD

**Discussion by ppo** [withastro/roadmap#985] [Proposal]
### Body

# Summary

Add a `--no-typescript` flag.  

# Background & Motivation

Before only `--template` was available. Now everything except "no typescript"… seriously!?!  
Not to be a hater but take a step back and think about that. You did an amazing job with Astro in general… but then you miserably fail on a stupid create script!?! How is that possible? 🤷‍♂️  Isn't obvious that a script that asks questions should allow to provide the answers directly in the call (to be scriptable)?

# Goals

Basically, allow to create a new Astro project with a single command defining all the parameters, and therefore without any human interaction needed.

# Example

```
pnpm create astro@latest foo -- --template minimal --no-typescript --install --git --yes --skip-houston
```

**Comment by Princesseuh** [comment]
Doing `pnpm create astro --typescript relaxed` or `--typescript base` achieve what you describe

**Comment by Fryuni** [comment]
Link to related issue comment: https://github.com/withastro/astro/issues/11609#issuecomment-2267344751

**Comment by ppo** [comment]
OK, so it's rather a wording problem. I'm not a native English speaker but I understand "Do you plan to write TypeScript?" like answering no means I don't want to use TypeScript and therefore I expect that it will use a non-TypeScript template, so I guess a JavaScript template.

That said, as @Fryuni said in #11609:

> Astro always uses TypeScript. No matter what you chose your code will be processed using a TS compiler.

So I think the question could be improved. Maybe asking for the strictness level and mentioning "choose relaxed if you don't plan to write TS".
